string scene_object_name
float32[] scene_object_pose